mlflow/types/llm.py

```diff
@@ -90,24 +127,73 @@ def from_dict(cls, data):
         return cls(**filtered_data)
 
 
+@dataclass
+class FunctionToolCallArguments(_BaseDataclass):
+    """
+    The arguments of a function tool call made by the model.
+
+    Args:
+        arguments (str): A JSON string of arguments that should be passed to the tool.
+        name (str): The name of the tool that is being called.
+    """
+
+    name: str
+    arguments: str
+
+    def __post_init__(self):
+        self._validate_field("name", str, True)
+        self._validate_field("arguments", str, True)
+
+    def to_tool_call(self, id):
+        return ToolCall(id=id, function=self)
```

Comment:
This is a convenience method so that users don't have to manually construct a `ToolCall` class. At the moment, the `ToolCall` class is a necessary but useless wrapper for the actual tool call arguments.